docs/api/index.rst

```diff
@@ -11,7 +11,14 @@ API
 Changelog
 ---------
 
-1.14 (Unreleased)
+1.15 (Unreleased)
+'''''''''''''''''
+- ``If-Match`` and ``If-None-Match`` precondition headers now check if ``ETag`` is
+  equal or not equal to the provided value (as in RFC 2616). Previous versions
+  asserted then to be greater or lower.
```

Comment:
I agree with @leplatrem's previous comment about this verbiage. How about `If-Match and If-None-Match precondition headers now check the ETag for strict equality. Previous versions would allow requests if they seemed to be more recent than the current version.`